agent/spec/lib/kontena/workers/node_stats_worker_spec.rb

```diff
@@ -138,12 +129,13 @@
   describe '#publish_node_stats' do
 
     before(:each) do
+      subject.instance_variable_set('@node', node)
```

Comment:
The specs were already doing `subject.instance_variable_set("@statsd", statsd)` for the `send_statsd_metrics`... refactored the worker to have a `configure` method, and call that in the rspec context.